tests/manage/test_dynamic_pvc_accessmodes_with_reclaim_policies.py

```diff
@@ -0,0 +1,295 @@
+        assert helpers.wait_for_resource_state(
+            resource=pod_obj2, state=constants.STATUS_PENDING
+        )
+        self.verify_expected_failure_event(
+            ocs_obj=pod_obj2, failure_str=self.expected_pod_failure
+        )
+
+        pod_obj1.delete()
+        pod_obj1.ocp.wait_for_delete(resource_name=pod_obj1.name)
+
+        # Wait for second pod to be in Running state
+        assert helpers.wait_for_resource_state(
+            resource=pod_obj2, state=constants.STATUS_RUNNING, timeout=240
+        )
+
+        assert pod.verify_data_integrity(
+            pod_obj=pod_obj2, file_name=file_name,
+            original_md5sum=md5sum_pod1_data
+        )
+
+        pod_obj2.run_io(
+            storage_type=self.storage_type, size=self.io_size, runtime=30,
+            fio_filename=pod_obj2.name
+        )
+        pod.get_fio_rw_iops(pod_obj2)
+
+        # Again verify data integrity
+        assert pod.verify_data_integrity(
+            pod_obj=pod_obj2, file_name=file_name,
+            original_md5sum=md5sum_pod1_data
+        )
+
+        pod_obj2.delete()
+        pod_obj1.ocp.wait_for_delete(resource_name=pod_obj2.name)
+
+
+@pytest.mark.skip("TODO")
+@tier1
+@pytest.mark.usefixtures(
+    create_ceph_block_pool.__name__,
+    create_rbd_secret.__name__,
+    create_cephfs_secret.__name__,
+    create_project.__name__
+)
+@pytest.mark.parametrize(
+    argnames=["interface_type", "reclaim_policy"],
+    argvalues=[
+        pytest.param(
+            *[constants.CEPHBLOCKPOOL, constants.RECLAIM_POLICY_RETAIN],
+            marks=pytest.mark.polarion_id("OCS-547")
+        ),
+        pytest.param(
+            *[constants.CEPHBLOCKPOOL, constants.RECLAIM_POLICY_DELETE],
+            marks=pytest.mark.polarion_id("OCS-538")
+        ),
+        pytest.param(
+            *[constants.CEPHFILESYSTEM, constants.RECLAIM_POLICY_RETAIN],
+            marks=pytest.mark.polarion_id("OCS-542")
+        ),
+        pytest.param(
+            *[constants.CEPHFILESYSTEM, constants.RECLAIM_POLICY_DELETE],
+            marks=pytest.mark.polarion_id("OCS-529")
+        )
+    ]
+)
+class TestRWXDynamicPvc(BaseDynamicPvc):
+    """
+    Automates the following test cases
+    OCS-547 - RBD Based RWX Dynamic PVC creation with Reclaim policy set
+    to Retain
+    OCS-538 - RBD Based RWX Dynamic PVC creation with Reclaim policy set
+    to Delete
+    OCS-542 - CephFS Based RWX Dynamic PVC creation with Reclaim policy set
+    to Retain
+    OCS-529 - CephFS Based RWX Dynamic PVC creation with Reclaim policy set
+    to Delete
+    """
+    access_mode = constants.ACCESS_MODE_RWX
+
+    @pytest.fixture()
+    def setup_base(self, request, interface_type, reclaim_policy):
+
+        def finalizer():
+            self.cleanup()
+        request.addfinalizer(finalizer)
+
+        self.dynamic_pvc_base(interface_type, reclaim_policy)
+
+    def test_rwx_dynamic_pvc(self, setup_base):
```

Comment:
let's remove the test_ from the function name so this case won't run at all and report to Polarion wrong results

Reply:
Done

`========================================================== test session starts ==========================================================
platform linux -- Python 3.7.3, pytest-5.0.1, py-1.8.0, pluggy-0.12.0
rootdir: /home/sagrawal/Projects/github/ocs/ocs-ci-fork, inifile: pytest.ini
plugins: html-1.21.1, reportportal-1.0.5, metadata-1.8.0, logger-0.5.0, marker-bugzilla-0.9.1.dev2
collected 4 items                                                                                                                       
<Package /home/sagrawal/Projects/github/ocs/ocs-ci-fork/tests/manage>
  <Module test_dynamic_pvc_accessmodes_with_reclaim_policies.py>
    <Class TestRWODynamicPvc>
        <Function test_rwo_dynamic_pvc[CephBlockPool-Retain]>
        <Function test_rwo_dynamic_pvc[CephBlockPool-Delete]>
        <Function test_rwo_dynamic_pvc[CephFileSystem-Retain]>
        <Function test_rwo_dynamic_pvc[CephFileSystem-Delete]>
`